.gitignore

```diff
@@ -42,4 +42,6 @@ doc/protocol-binary.txt
 /doc/doxy
 /memcached.spec
 .*.swp
+.swp
```

Comment:
> A gitignore file specifies intentionally untracked files that Git should ignore.

The software itself should not care about what editor you are using.  
As you know, vim is indeed a common editor, but other people prefer emacs, nano, etc. It's difficult to define _common_.  
Obviously, we cannot add editors' ignored pattern here.

Reply:
Reasonable. I thought `.*.swp` was already there for the same reason.